src/main/java/ox/app/game/Board.java

```diff
@@ -23,12 +28,12 @@ private Board(int width, int height) {
         this.width = width;
         this.height = height;
         this.size = width * height;
+
     }
 
     public void placeSymbol(Coordinates coordinates, Symbol symbol) {
```

Comment:
You can think about delegating responsiblity of stuff connected with Map handling to another class